week1/homework/HackYourWeather1/hy-weather1/src/components/CityTitle/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function CityTitle(props) {
+    return (
+        <h2 style={{ textAlign: 'left', marginLeft: '1.5rem' }}>{props.name}, {props.country}</h2>
+    )
+}
```

Comment:
Just suggesting an alternative here. 
```suggestion
const CityTitle = (props) => (
    <h2 style={{ textAlign: 'left', marginLeft: '1.5rem' }}>{props.name}, {props.country}</h2>
)
```